os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c

```diff
@@ -123,6 +123,16 @@ void pg_timer_int_handler(void *Data)
 	pm_activity(PM_IDLE_DOMAIN, 9);
 }
 
+void pg_lock_timer_int_handler(void)
```

Comment:
Instead of having 2 copies, we can have add some condition to check whether we are deploying timer1 or timer2, we can do that by expanding struct pm_timer_wakup_s